cli.py

```diff
@@ -11,19 +11,29 @@ def check_exit(user_input):
 
 
 def cli_input():
-    user_input = input("> ")
-    if check_exit(user_input):
-        return None
-    return user_input
+    try:
+        user_input = input("> ")
+    except (KeyboardInterrupt, EOFError): # We can add a print command here if you'd like
+        pass
```

Comment:
I see what you mean, empty `print()` would definitely make output nicer.

I like the idea. In addition, explicit `return None` after that would make very clear that it is going to quit after that.

Don't forget to remove the comment after that.

Reply:
Perfect. I'll modify the code and push it again.